linux/files/sudoerg

```diff
@@ -0,0 +1,18 @@
+# managed by salt
```

Comment:
Check files/hosts or resolv.conf for nicer header about managed by salt and do not edit.